### DISCUSSION TITLE
기술스택 카테고리 변수명 통일

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#2] [Ideas]
# 기술스택 카테고리 변수명 통일

**Comment by Hyevvy** [comment]
# 프론트엔드
Html5
TypeScript
React
Next.js
Vue
Nuxt.js
Svelte
Gatsby
Angular
Sass
Redux
Recoil
Bootstrap
Mui
Tailwind

**Comment by Hyevvy** [comment]
# back
Node.js
Express
FastAPI
GraphQL
Oracle
NestJS
MongoDB
MySQL
PostgreSQL
Firebase
Appwrite
Django
Spring

**Comment by Hyevvy** [comment]
# ios
Swift
Flutter
RN

# android
Java
Kotlin
Flutter
RN

**Comment by Hyevvy** [comment]
# designer
Figma
Zeplin

**Comment by NamgyungKim** [comment]
# AI
Python
java
C/C++
TensorFlow.js
R
Lua
Julia
Swift

**Reply by YuHyun-P** [reply]
변수명은 C/C++로 하나이지만 팀 Figma에서는 C와 C++ 기술 뱃지가 2개라서 어떻게 할지 정해야 합니다!
![image](https://user-images.githubusercontent.com/96400112/173226593-5317139b-0075-4ad4-ae17-dd33477601d2.png)

- 제 의견은 C/C++ 하나로 묶어서 많이 사용하는 걸 봐서 변수명은 그대로, 기술뱃지는 C나 C++ 중 하나로 선택해서 가면 될 것 같습니다

**Reply by Hyevvy** [reply]
남경님 이 부분 AI => ai로 수정부탁드립니당!!

**Comment by YuHyun-P** [comment]
✅ C/C++ 하나로 묶고 기술뱃지는 C++로 결정
